aptos-move/framework/aptos-framework/sources/configs/Stake.move

```diff
@@ -117,16 +113,35 @@ module AptosFramework::Stake {
         config: ValidatorConfig,
     }
 
+    /// Validator set configurations that will be stored with the @CoreResources account.
+    struct ValidatorSetConfiguration has key {
+        // A validator needs to stake at least this amount to be able to join the validator set.
+        // If after joining the validator set and at the start of any epoch, a validator's stake drops below this amount
+        // they will be removed from the set.
+        minimum_stake: u64,
+        // A validator can only stake at most this amount. Any larger stake will be rejected.
+        // If after joining the validator set and at the start of any epoch, a validator's stake exceeds this amount,
+        // they will be removed from the set.
+        // TODO: Revisit whether a validator should be removed from the validator set if their stake exceeds the max.
```

Comment:
Why wouldn't you just limit the amount of stake rather than kick them out of the validator set?

Reply:
I think this is a weird behavior as well. It's something existing that I'm thinking whether it still makes any sense. Limiting the stake is not the perfect solution either as it'd stop people who hit the max from getting rewards